.coveragerc

```diff
@@ -0,0 +1,36 @@
+[run]
+branch = True
+include=
+    # Include the following source code folders
+    **/lambdas/**
+    **/tests/**
+
+[report]
+exclude_lines=
+    # Have to re-enable the standard pragma
+    pragma: no cover
+
+    # Don't complain about missing debug-only code:
+    def __repr__
+    if self\.debug
+
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
```

Comment:
Why are we excluding these?

Reply:
"Let's delete this and see what happens" ~ Aaron

Reply:
Agreed